react/package.json

```diff
@@ -5,79 +5,81 @@
   "homepage": "",
   "dependencies": {
     "@apollo/client": "^3.6.9",
-    "@auth0/auth0-react": "^1.10.2",
+    "@auth0/auth0-react": "^1.11.0",
+    "@babel/runtime": "^7.19.0",
     "@chakra-ui/icons": "^1.1.7",
-    "@chakra-ui/react": "^1.6.10",
-    "@emotion/react": "^11",
-    "@emotion/styled": "^11",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@chakra-ui/react": "^1.8.9",
+    "@emotion/react": "^11.10.4",
```

Comment:
figure out what this is for.

Reply:
Chakra UI is what @spaudanjo and @flisowna use for React components.

Reply:
I have checked [Chakra UI](https://chakra-ui.com/getting-started) and @emotion/react, @emotion/styled, and framer-motion are its dependencies